src/dex-ui/services/HederaService/HederaService.ts

```diff
@@ -69,11 +70,11 @@ function createHederaService() {
       signer,
     } = addLiquidityDetails;
 
-    // TODO: remove fallbacks if no signer and not all details are provided
```

Comment:
I'm receiving an **INSUFFICIENT_PAYER_BALANCE** error when attempting to add liquidity with an HBAR pair. I have HBARs in my wallet.

![Screen Shot 2023-01-12 at 6 53 30 PM](https://user-images.githubusercontent.com/54907098/212205688-31212f3e-969c-4ba0-8b1e-0ce736bea1e5.png)

Reply:
when we swap other tokens for HBAR, we have to check the contract call and verify the HBAR amount we have received since only `Crypto Transfer` can be seen for Token Transfer its working fine for normal Tokens Swap, we can get the details of HBAR received in first Contract Call Log.

May be we can discuss more on call, I had a call with team to understand it.